src/troute-routing/troute/routing/diffusive_utils.py

```diff
@@ -1231,17 +1379,17 @@ def diffusive_input_data_v02(
         qlat,
         qlat_g,
     )
-
+    
     # ---------------------------------------------------------------------------------
     #                              Step 0-8
 
     #       Prepare upstream boundary (top segments of head basin reaches) data
     # ---------------------------------------------------------------------------------
-    ds_seg = []
-    upstream_flow_array = np.zeros((len(ds_seg), nsteps+1)) # <- this is a place holder until we figure out what to do with upstream boundary 
-    nts_ub_g = int((tfin_g - t0_g) * 3600.0 / dt_ub_g)
-    ubcd_g = fp_ubcd_map(frnw_g, pynw, nts_ub_g, nrch_g, ds_seg, upstream_flow_array)
-
+    #ds_seg = []
+    #upstream_flow_array = np.zeros((len(ds_seg), nsteps+1)) # <- this is a place holder until we figure out what to do with upstream boundary     
+    nts_ub_g = int((tfin_g - t0_g) * 3600.0 / dt_ub_g)  
+    #ubcd_g = fp_ubcd_map(frnw_g, pynw, nts_ub_g, nrch_g, ds_seg, upstream_flow_array)
```

Comment:
should really consider renaming these variables so someone has a chance of understanding what they are/mean

Reply:
agreed.  I will do it gradually.  Or place comments on the meaning of variables?